relayer-cli/Cargo.toml

```diff
@@ -26,11 +26,11 @@ telemetry   = ["ibc-relayer/telemetry", "ibc-telemetry"]
 rest-server = ["ibc-relayer-rest"]
 
 [dependencies]
-ibc              = { version = "0.12.0", path = "../modules", features = ["std", "clock"] }
-ibc-relayer      = { version = "0.12.0", path = "../relayer" }
+ibc              = { version = "0.13.0-rc.0", path = "../modules", features = ["std", "clock"] }
+ibc-relayer      = { version = "0.13.0-rc.0", path = "../relayer" }
 ibc-proto        = { version = "0.16.0", path = "../proto" }
```

Comment:
Decided not to bump the proto version because release changes don't touch the proto crate.